src/parameters.toml

```diff
@@ -763,3 +763,241 @@ alias = "avogad"
+type = "float"
+explanation = "Minimum area fraction per updraft. Parameter not described in the literature."
+
+[mixing_length_eddy_viscosity_coefficient]
+alias = "tke_ed_coeff"
+value = 0.14
+type = "float"
+explanation = "Turbulence kinetic energy diffusivity coefficient for the EDMF mixing length closure; denoted c_m. See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1."
+
+[mixing_length_diss_coeff]
+alias = "tke_diss_coeff"
+value = 0.22
+type = "float"
+explanation = "Turbulence kinetic energy dissipation coefficient for the EDMF mixing length closure; denoted c_d. See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1."
+
+[mixing_length_static_stab_coeff]
+alias = "static_stab_coeff"
+value = 0.4
+type = "float"
+explanation = "Static stability coefficient for the EDMF mixing length closure; denoted c_b. See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1."
+
+[mixing_length_tke_surf_scale]
+alias = "tke_surf_scale"
+value = 3.75
+type = "float"
+explanation = "Ratio of turbulence kinetic energy to squared friction velocity in the surface layer for the EDMF mixing length closure; denoted κ_*². See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1. Note: the square root, i.e. κ_*, is listed in the reference."
+
+[mixing_length_Prandtl_number_scale]
+alias = "Prandtl_number_scale"
+value = 4.076923076923077
+type = "float"
+explanation = "Cospectral budget factor for turbulent Prandtl number for the EDMF mixing length closure, denoted ω_pr. In Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Eq. 36, it is described as a phenomenological constant, denoted by ω₂ and set to 53/13 ≈ 4.0769..."
+
+[mixing_length_Prandtl_number_0]
+alias = "Prandtl_number_0"
+value = 0.74
+type = "float"
+explanation = "Turbulent Prandtl number in neutral conditions; denoted Pr_{t,0}. See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1 and Eq 36."
+
+[mixing_length_Ri_crit]
+alias = "Ri_crit"
+value = 0.25
+type = "float"
+explanation = "Critical gradient Richardson number. It is an upper limit to the gradient Richardson number . See Li (2019) [https://doi.org/10.1016/j.atmosres.2018.09.015], Section 6.2 for details"
+
+[mixing_length_smin_ub]
+alias = "smin_ub"
+value = 0.1
+type = "float"
+
+[mixing_length_smin_rm]
+alias = "smin_rm"
+value = 1.5
+type = "float"
+
+[mixing_length_l_max]
+alias = "l_max"
+value = 1.0e6
+type = "float"
+
+[entrainment_factor]
+alias = "entrainment_factor"
+value = 0.13
+type = "float"
+explanation = "Scaling constant for entrainment rate. See Cohen et al. (2022) Table 2 & Eq 39, 40 [https://doi.org/10.1029/2020MS002162]"
+
+[detrainment_factor]
+alias = "detrainment_factor"
+value = 0.51
+type = "float"
+explanation = "Scaling constant for detrainment rate. See Cohen et al. (2022) Table 2 & Eq 39, 40 [https://doi.org/10.1029/2020MS002162]"
+
+[entrainment_massflux_div_factor]
```

Comment:
This parameter was used for the plume test case in TC.jl, and has been removed as far as I can tell.

Reply:
It's defined [here ](https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/TurbulenceConvection/types.jl#L447)as ``c_div`` and looks like it's being used for [εδModelParams](https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/TurbulenceConvection/types.jl#L515-L527).  Not sure if we should remove it.

Reply:
Deleted